ci/tasks/e2e-test.sh

```diff
@@ -2,12 +2,7 @@
 
 set -e
 
-yarn install
-yarn playwright install-deps
-yarn playwright install
-
 export GIT_COMMIT=`git rev-parse HEAD`
 mkdir -p playwright-report/$APP_ENV/$GIT_COMMIT
 
-yarn create-test-users
-yarn playwright test
+npm exec playwright test
```

Comment:
Is yarn not installed in the playwright image? I'd be in favor of dumping yarn for npm across the platform.

Reply:
yarn is in the playwright image, but this works either way. I'm ambivalent about the switch (separate issue)